src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  static died() {
+    Animal.alive = Animal.alive.filter(({ health }) => health > 0);
+  }
+
+  health = 100;
+
+  constructor(name) {
```

Comment:
You should allow the creation of animals with different  "health", 100 just by default if not passed. And create a constant for this value as it's a magic value in this case